tests/test_main.cpp

```diff
@@ -79,17 +81,20 @@ void init_global_game_state( const std::vector<std::string> &mods )
     world_generator->init();
     WORLDPTR test_world = world_generator->make_new_world( mods );
     assert( test_world != NULL );
-    world_generator->set_active_world(test_world);
+    world_generator->set_active_world( test_world );
     assert( world_generator->active_world != NULL );
 
     g->load_core_data();
     g->load_world_modfiles( world_generator->active_world );
 
     g->u = player();
-    g->u.create(PLTYPE_NOW);
+    g->u.create( PLTYPE_NOW );
 
     g->m = map( get_option<bool>( "ZLEVELS" ) );
 
+    std::vector<overmap_special_placement> empty_specials;
+    overmap_buffer.create_custom_overmap( g->get_levx(), g->get_levy(), empty_specials );
```

Comment:
I'm pretty sure those x/y arguments are of wrong "type": `create_custom_overmap` passes its arguments to overmap constructor, meaning those should be in overmap coords (as in, one coord - one overmap).
But `get_levx/y` are in submap coords (as in, one coord, one submap).